bin/check_samplesheet.py

```diff
@@ -92,9 +92,9 @@ def check_samplesheet(file_in, file_out):
                 if fasta:
                     if fasta.find(" ") != -1:
                         print_error("FastA file contains spaces!", "Line", line)
-                    if not fasta.endswith(".fasta.gz") and not fasta.endswith(".fa.gz") and not fasta.endswith(".fna.gz") and not fasta.endswith(".fna"):
+                    if not fasta.endswith(".fasta.gz") and not fasta.endswith(".fa.gz") and not fasta.endswith(".fna.gz")  and not fasta.endswith(".fasta") and not fasta.endswith(".fa") and not fasta.endswith(".fna"):
```

Comment:
Could change this to:

```suggestion
                    if Path(fasta).suffixes.join("") not in (".fasta.gz", ".fa.gz", ".fna.gz", ".fasta", ".fa", ".fna"):
```

Reply:
Was just following standard template, could do but make check with Harshil that there wasn't a specific reason why he didn't the  library that Path is from already